schema/rpc.fbs

```diff
@@ -560,3 +563,112 @@ table SaveFileNotification {
     /// Recommended filename
     expected_filename: string;
 }
+
+// ------------ Firmware tool -------------
```

Comment:
although we didn't do this in the rest of the rpc.fbs file, I think this is a good opportunity to start organizing things in files. Can you pull the firmware tool section into its own file in an rpc folder?